src/api/spec/models/package_spec.rb

```diff
@@ -440,12 +440,22 @@
       before { stub_request(:post, backend_url) }
 
       it { is_expected.to be_truthy }
+
+      it "without error" do
```

Comment:
That's no excuse to not do it properly:-) 'it without error' isn't very speaking. Maybe 'it has no errors' ? Or 'it returns no errors'?

Now that you say it... why do you need to call subject here? Seems you are testing package.errors.details in the end.